packages/react-i18n/src/test/utilities.test.tsx

```diff
@@ -398,4 +399,18 @@ describe('translate()', () => {
       });
     });
   });
+
+  describe('wrapWithDisclaimer', () => {
+    const translations = {
+      disclaimer: {language: {en: '{content}', de: '{content} (in German)'}},
+    };
+    it('wraps the content in the translation', () => {
+      expect(wrapWithDisclaimer('Learn more', 'en', translations, locale)).toBe(
+        'Learn more',
+      );
+      expect(wrapWithDisclaimer('Learn more', 'de', translations, locale)).toBe(
+        'Learn more (in German)',
```

Comment:
If we can access the translation to add `In German` could we not update the string itself?